Not all trait items implemented, missing: `collect_str`

When trying to install pythonize I run into the error:


```bash
    Blocking waiting for file lock on build directory
   Compiling pyo3 v0.12.4
   Compiling pyo3 v0.12.4 (https://github.com/PyO3/pyo3#92b7a973)
   Compiling pythonize v0.12.0
 error[E0046]: not all trait items implemented, missing: `collect_str`
  --> /home/***/.cargo/registry/src/github.com-1ecc6299db9ec823/pythonize-0.12.0/src/ser.rs:49:1
   |
49 | impl<'py> ser::Serializer for Pythonizer<'py> {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `collect_str` in implementation
   |
   = help: implement the missing item: `fn collect_str<T>(self, _: &T) -> std::result::Result<<Self as Serializer>::Ok, <Self as Serializer>::Error> where T: std::fmt::Display { todo!() }`
```

What could be the source of this problem?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+## Unreleased
+ - Require `std` feature of `serde`.
+
 ## 0.12.0 - 2020-11-22
 - Change release versioning to match `pyo3` major/minor version.
 - Implement `depythonizer`

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,10 +11,10 @@ documentation = "https://docs.rs/crate/pythonize/"
 
 
 [dependencies]
-serde = { version = "1.0", default-features = false }
+serde = { version = "1.0", default-features = false, features = ["std"] }
 pyo3 = { version = "0.12", default-features = false }
 
 [dev-dependencies]
-serde = { version = "1.0", features = ["derive"] }
+serde = { version = "1.0", default-features = false, features = ["derive"] }
 serde_json = "1.0"
 maplit = "1.0.2"

